runtime-testsuite/test/org/antlr/v4/test/runtime/cpp/CppRuntimeTests.java

```diff
@@ -0,0 +1,11 @@
+package org.antlr.v4.test.runtime.cpp;
+
+import org.antlr.v4.test.runtime.RuntimeTests;
+import org.antlr.v4.test.runtime.RuntimeRunner;
+
+public class CppRuntimeTests extends RuntimeTests {
```

Comment:
Not sure we want to remove all these test entry points. It was very easy to select and run individual tests even if having the duplicated code around was "bad".

Reply:
Actually the previous tests were also being generated and it was not very convenient to select individual tests. You had to generate them at first. With the current system both groups and individual tests are being generated automatically:

![image](https://user-images.githubusercontent.com/1150330/174402365-369686c6-85ba-47b9-94db-8cce71dde4fd.png)

With the dynamic tests there are no dummy `testOne` methods.

Reply:
heh, that does look a lot better! And also removes code which is fantastic!